std/algorithm/package.d

```diff
@@ -170,8 +170,8 @@ sort(a);                   // no predicate, "a < b" is implicit
 
 Macros:
 WIKI = Phobos/StdAlgorithm
-SUBMODULE = $(LINK2 std_algorithm_$1.html, std.algorithm.$1)
-SUBREF = $(LINK2 std_algorithm_$1.html#.$2, $(TT $2))$(NBSP)
+SUBMODULE = $(MREF std, algorithm, $1)
+SUBREF = $(REF_ALTTEXT $2, $2, std, algorithm, $1)
```

Comment:
It was kind of by accident, but I'd say it should be added globally to the REF macros anyway, probably using `<code>` instead of `<tt>`.